config/locales/en.yml

```diff
@@ -20,4 +20,160 @@
+      description: "Improving your developer culture is too important to be left unplanned."
+      encourage:
+        title: "Set aside time"
+        description: "Each Friday, encourage your managers, employees, and coworkers to spend at least two hours working on the open source projects your organization already uses."
+      policy:
+        title: "Make it a habit"
+        description: "Create clear, internal policies so employees can easily and consistently contribute to open source."
+
+  # app/views/home/maintainers.html.erb
+  maintainers:
+    title: "Build software with help from others"
+    description:  "Every Friday, give your software project some love and make it easier for others to contribute."
+    involve:
+      title: "Help people discover your project"
+      description: "Creating a project is only the beginning. Why not spread the&nbsp;word?"
+      workload:
+        title: "Reduce your workload:"
+        description_html: "Ensure your project's README file details what your project does, its scope and contribution needs, and how to contribute to < a href=\"%{url}\">leverage your community</a>."
+      promote:
+        title: "Promote your work"
+        description_html: "Consider creating a Twitter account, blog posts and website to <a href=\"%{url}\">help people find and follow your project</a>."
+    growing:
+      title: "Grow your community"
+      description: "Everybody was new once. Ease someone else's first-time contribution jitters."
+      document:
+        title: "Document everything"
+        description_html: "<a href=\"%{survey}\">GitHub's 2017 Open Source Survey</a> showed incomplete or confusing documentation is the biggest problem for open source users. Ensure you are <a href=\"%{guide}\">documenting your processes </a>  for how to contribute to your project."
+      help:
+        title: "Get help"
+        Look_out_html: "Look out for promising potential contributors or maintainers to <a href=\"%{url}\"> share ownership of your project.</a> Make an effort to welcome first-time contributors in particular."
+    workflow:
+      title: "Improve your workflow"
+      description: "Learn from other maintainers on how to be more effective."
+      boundaries:
+        title: "Set boundaries"
+        description_html: "Decide on what the scope of the project is and <a href=\"%{pause}\">set clear boundaries for yourself</a>. Saying no <a href=\"%{sayno}\"> is as important as saying yes</a>."
+      automation:
+        title: "Automation"
+        description_html: "<a href=\"%{url}\">Bring in the robots</a> to automate as much as possible through tests and other checks to save yourself time."
+  visitor:
+    title: "Get ready for Open Source Friday"
+    description: "Spend time now to save time later."
+
+  endorsement:
+    title: "You're in good company" 
+    Kim_Crayton:
+      title_html: 'Kim Crayton (%{url}),<br>Community Engineer, Writer, Founder'
```

Comment:
Maybe worth splitting this into two sections so they don't each need the `<br>` embedded in them?

Reply:
It cannot split two section because japansese translation has reverse order reflecting its culture.
 EN:    name,<br> title  organization
 JA:    organization,<br> title, name

Reply:
Apologies for the delay, I've been out on paternity leave. I think we should still have title first even in Japan because some of these folks aren't speaking "officially" on behalf of their organisation.